File: Data Structures & Algorithms/duplicate-integer/submission-0.cpp
class Solution {
public:
    bool hasDuplicate(vector<int>& nums) {
        int number = 0;
        int counter = 0;
        for(int i = 0; i < nums.size(); i++){
            number = nums[i];
            counter = 0;
            for(int j = 0; j < nums.size(); j++){
                if(number == nums[j]){
                counter++;
                }
            }
            if(counter > 1){
                return true;
            }
            
        }
        return false;
    }
};

File: Data Structures & Algorithms/longest-consecutive-sequence/submission-0.py
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
        tempSet = sorted(set(nums))
        if not tempSet:
            return 0

        counter = 1
        length = 1

        for i in range(0, len(tempSet)-1):
            if tempSet[i+1] -1 == tempSet[i]:
                length += 1
            else:
                length = 1

            if length > counter:
                counter = length

        return counter